project_profile_counter.py

```diff
@@ -0,0 +1,28 @@
+import json
+import requests
+import collections
+
+SERVER = 'https://tokyo.sdelements.com'
+API_TOKEN = 'f258c47557a3f98f55d4fbd0cb9d8354c86fbb52'
+PROJECTS_URL = '{server}/api/v2/projects/?page_size=100'.format(server=SERVER)
+REQUESTS_HEADER = {
+    'Authorization': "Token " + API_TOKEN,   # Authorization header + generated API TOKEN
+    'Accept': "application/json"             # Accepts API response in JSON format
+}
+
+url = PROJECTS_URL
+profiles = []
+while True:
+    response = requests.get(url, headers=REQUESTS_HEADER) 
+    data = response.json()
+
+    for item in data['results']:    # loop through the values in the results key 
```

Comment:
No point writing comments that repeat what the code is saying. Comments should say why you are doing, or something that might not be obvious when reading the code.